frontend/pages/ManageControlsPage/SetupExperience/cards/SetupAssistant/components/AdvancedOptionsForm/_styles.scss

```diff
@@ -4,7 +4,9 @@
   gap: $pad-large;
 
   &__accordion-title {
-    // use this so we dont center the text.
-    justify-content: normal;
+    // Immediate children of a flexbox, which this element is, stretch content by default. There are a few ways to
+    // cancel that behavior (another would be to set width: fit-content), but we need one of them here to make sure
+    // the reveal button isn't wider than it needs to be.
+    align-self: flex-start;
```

Comment:
👍 

Mind adding this to the [AddInstallSoftware stylesheet](https://github.com/fleetdm/fleet/blob/34530-setup-experience-style-updates/frontend/pages/ManageControlsPage/SetupExperience/cards/InstallSoftware/components/AddInstallSoftware/_styles.scss) (under `.add-install-software {`) for consistency? (I missed it in an earlier PR and the spacing is wrong)

```scss
  &__macos_options_form {
    display: flex;
    flex-direction: column;
    gap: $pad-medium;

    button.reveal-button {
      align-self: flex-start;
    }
  }
```